pages/linux/archlinux-java.md

```diff
@@ -0,0 +1,19 @@
+# archlinux-java
+
+> A helper script that provides functionalities for Java environments.
+
+- List Java environments currently installed:
+
+`archlinux-java status`
+
+- Change the default Java environment:
+
+`archlinux-java set {{java_env_name}}`
```

Comment:
I think this variable has a better name:
```suggestion
`archlinux-java set {{java_environment}}`
```
Because `java_environment` is defined in the previous example.